docs/helm-deployment-aws_cloud.md

```diff
@@ -16,6 +18,11 @@ To run the Alfresco Content Services (ACS) deployment on AWS with Kubernetes req
 
 **Note:** You don't need to clone this repository to deploy Alfresco Content Services.
 
+### Prepare and configure Kops
+
+Kops requires certain configurations on your AWS account. Please follow the guide
+[here](https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-your-environment)
+through "Creating your first cluster". We will create the cluster for ACS in the next step.
```

Comment:
Probably a better wording would be "The cluster for ACS will be created in the next step".